typescript/expCalc.ts

```diff
@@ -0,0 +1,81 @@
+function Add(x,y)
+{
+	return x+y;
+}
+
+function Subtract(x,y)
+{
+	return x-y;
+}
+
+function Multiply(x,y)
+{
+	return x*y;
+}
+
+function TryDivide(x,y)
+{
+	if(x >= y)
+		return x/y;
+	else
+		return 0;
+}
+
+function getOperands(cleanInputString)
+{
+	let numArray: number[] = [];
+	for(let i in cleanInputString)
+	{
+		numArray[i] = parseFloat(cleanInputString[i]);
```

Comment:
Please handle parse errors.  For example, 2.3 will succeed, however, 2..3 will fail.